packages/grid/src/vaadin-grid.js

```diff
@@ -892,7 +907,8 @@ class Grid extends ElementMixin(
 
     this._updateRowState(row, 'first', index === 0);
     this._updateRowState(row, 'last', index === this._effectiveSize - 1);
-    this._updateRowState(row, 'odd', index % 2);
+    this._updateRowAndCells(row, 'odd', index % 2);
+    this._updateRowAndCells(row, 'even', index % 2 === 0);
```

Comment:
Not sure about adding the `even` part. Someone may also want to style cells on an unexpanded (collapsed) row or unselected row but we're probably not adding those either.

Reply:
Well, maybe having `even` actually still makes sense

Reply:
I added `even` mainly to cover the row stripes variant use case, as we can't use `:not(::part(odd-cell))`:

https://github.com/vaadin/web-components/blob/5f632380cae5457e29b3b12485fd95245578b044/packages/grid/theme/lumo/vaadin-grid-styles.js#L312-L313